libmachine/mcnutils/utils.go

```diff
@@ -23,34 +22,6 @@ func (e MultiError) Error() string {
 	return aggregate
 }
 
-// GetHomeDir returns the home directory
```

Comment:
As it says this is used only for some drivers and we may be not using those drivers so better to remove 👍